docs/linters.md

```diff
@@ -555,6 +556,38 @@ If you prefer not to suggest fixes for pointers in required fields, you can chan
 If you prefer not to suggest fixes for `omitempty` in required fields, you can change the `omitempty.policy` to `Warn` or `Ignore`.
 If you prefer not to suggest fixes for `omitzero` in required fields, you can change the `omitzero.policy` to `Warn` and also not to consider `omitzero` policy at all, it can be set to `Forbid`.
 
+## References
+
+The `references` linter ensures that field names use 'Ref'/'Refs' suffixes instead of 'Reference'/'References'.
+
+By default, `references` is enabled and enforces this naming convention. The linter checks that:
+- Fields ending with 'Reference' are suggested to use 'Ref' instead
+- Fields ending with 'References' are suggested to use 'Refs' instead
+
+### Configuration
+
+```yaml
+lintersConfig:
+  references:
+    allowRefAndRefs: false | true # Allow Ref/Refs suffixes for OpenShift compatibility. Defaults to `false`.
```

Comment:
We generally prefer enums over bools, so maybe lets change this to be more like `policy` config like we have in other places. Perhaps the values could be `AllowRefAndRefs` and `ForbidRefAndRefs`?

Reply:
Nit: Let's also drop the OpenShift mention. This is a vendor-neutral project.